src/lib/server/map.c

```diff
@@ -854,9 +854,10 @@ static int map_exec_to_vp(TALLOC_CTX *ctx, fr_pair_t **out, request_t *request,
 	int result;
 	char *expanded = NULL;
 	char answer[1024];
-	fr_pair_t **input_pairs = NULL;
-	fr_pair_t *output_pairs = NULL;
+	fr_pair_list_t *input_pairs = NULL;
+	fr_pair_list_t output_pairs;
 
+	fr_pair_list_init(&output_pairs);
```

Comment:
Probably want input_pairs to be initialised too?

Reply:
input_pairs is a pointer to a list header - then set to point to whichever list is required by radius_list() a few lines further down.